src/main/kotlin/com/exactpro/th2/lwdataprovider/db/CradleMessageExtractor.kt

```diff
@@ -296,5 +323,6 @@ private fun StoredGroupedMessageBatch.toShortInfo(): String = "${group}:${firstM
 
 data class CradleGroupRequest(
     val sort: Boolean,
+    val include: Set<ProviderMessageStream> = emptySet(),
```

Comment:
Why we can't execute all filtration via preFilter?

Reply:
We can. But it is used for filtration during pulling new messages. I didn't want to mix that logic with the new one (the prefilter is overwritten during pulling and we will have to somehow combine the filtration by streams with filtration required for pulling updates)

Reply:
@Nikita-Smirnov-Exactpro I can change it to be in preFilter if you need that

Reply:
Maybe split to two lambdas for each stage to use common approach for filtering?

Reply:
Done in next commits